### ISSUE TITLE
AlignmentAlgo=off: Boot loop, errors in log + wrong name on UI

### ISSUE THREAD

**Issue by Slider0007** [jomjol/AI-on-the-edge-device#1872]
### The Problem

- Activating new option Alignment off (PR #1867), system is throwing erros and boot looping.
- New option "off" is wrong labeled with "fast"

### Version

Latest rolling

### Logfile

```shell
[0d00h00m35s] 2023-01-19T09:37:48 <DBG> [MQTT IF] Published topic: water/WaterMeterTest/status, content: Aligning (msg_id=29)
[0d00h00m36s] 2023-01-19T09:37:49 <ERR> [ALIGN] Can't allocate AlignAndCutImage -> Exec this round aborted!
[0d00h00m36s] 2023-01-19T09:37:49 <DBG> [HEAP] ClassFlowAlignment-doFlow Heap Total: 1897602 | SPI Free: 1857179 | SPI Larg Block: 1835008 | SPI Min Free: 1004515 | Int Free: 40423 | Int Larg Block: 28160 | Int Min Free: 21067
[0d00h00m36s] 2023-01-19T09:37:49 <WRN> [CTRL] Fehler im vorheriger Schritt - wird zum 1. Mal wiederholt
[0d00h00m36s] 2023-01-19T09:37:49 <DBG> [MQTT IF] Published topic: water/WaterMeterTest/status, content: Aligning (msg_id=30)
[0d00h00m36s] 2023-01-19T09:37:50 <ERR> [ALIGN] Can't allocate AlignAndCutImage -> Exec this round aborted!
[0d00h00m36s] 2023-01-19T09:37:50 <DBG> [HEAP] ClassFlowAlignment-doFlow Heap Total: 1897594 | SPI Free: 1857179 | SPI Larg Block: 1835008 | SPI Min Free: 1004515 | Int Free: 40415 | Int Larg Block: 28160 | Int Min Free: 21067
[0d00h00m36s] 2023-01-19T09:37:50 <WRN> [CTRL] Fehler im vorheriger Schritt - wird zum 2. Mal wiederholt
[0d00h00m36s] 2023-01-19T09:37:50 <DBG> [MQTT IF] Published topic: water/WaterMeterTest/status, content: Aligning (msg_id=31)
[0d00h00m37s] 2023-01-19T09:37:50 <ERR> [ALIGN] Can't allocate AlignAndCutImage -> Exec this round aborted!
[0d00h00m37s] 2023-01-19T09:37:50 <DBG> [HEAP] ClassFlowAlignment-doFlow Heap Total: 1897586 | SPI Free: 1857179 | SPI Larg Block: 1835008 | SPI Min Free: 1004515 | Int Free: 40407 | Int Larg Block: 28160 | Int Min Free: 21067
[0d00h00m37s] 2023-01-19T09:37:50 <WRN> [CTRL] Fehler im vorheriger Schritt - wird zum 3. Mal wiederholt
[0d00h00m37s] 2023-01-19T09:37:50 <DBG> [MQTT IF] Published topic: water/WaterMeterTest/status, content: Aligning (msg_id=32)
[0d00h00m38s] 2023-01-19T09:37:51 <ERR> [ALIGN] Can't allocate AlignAndCutImage -> Exec this round aborted!
[0d00h00m38s] 2023-01-19T09:37:51 <DBG> [HEAP] ClassFlowAlignment-doFlow Heap Total: 1897594 | SPI Free: 1857179 | SPI Larg Block: 1835008 | SPI Min Free: 1004515 | Int Free: 40415 | Int Larg Block: 28160 | Int Min Free: 21067
[0d00h00m38s] 2023-01-19T09:37:51 <WRN> [CTRL] Fehler im vorheriger Schritt - wird zum 4. Mal wiederholt
[0d00h00m38s] 2023-01-19T09:37:51 <DBG> [MQTT IF] Published topic: water/WaterMeterTest/status, content: Aligning (msg_id=33)
[0d00h00m39s] 2023-01-19T09:37:52 <ERR> [ALIGN] Can't allocate AlignAndCutImage -> Exec this round aborted!
[0d00h00m39s] 2023-01-19T09:37:52 <DBG> [HEAP] ClassFlowAlignment-doFlow Heap Total: 1897586 | SPI Free: 1857179 | SPI Larg Block: 1835008 | SPI Min Free: 1004515 | Int Free: 40407 | Int Larg Block: 28160 | Int Min Free: 21067
[0d00h00m39s] 2023-01-19T09:37:52 <WRN> [CTRL] Fehler im vorheriger Schritt - wird zum 5. Mal wiederholt
[0d00h00m39s] 2023-01-19T09:37:52 <DBG> [MQTT IF] Published topic: water/WaterMeterTest/status, content: Aligning (msg_id=34)
[0d00h00m39s] 2023-01-19T09:37:53 <ERR> [ALIGN] Can't allocate AlignAndCutImage -> Exec this round aborted!
[0d00h00m39s] 2023-01-19T09:37:53 <DBG> [HEAP] ClassFlowAlignment-doFlow Heap Total: 1897594 | SPI Free: 1857179 | SPI Larg Block: 1835008 | SPI Min Free: 1004515 | Int Free: 40415 | Int Larg Block: 28160 | Int Min Free: 21067
[0d00h00m39s] 2023-01-19T09:37:53 <WRN> [CTRL] Fehler im vorheriger Schritt - wird zum 6. Mal wiederholt
[0d00h00m39s] 2023-01-19T09:37:53 <ERR> [CTRL] Wiederholung 5x nicht erfolgreich --> reboot
[0d00h00m39s] 2023-01-19T09:37:53 <INF> [OTA] Reboot triggered by Software (5s).
[0d00h00m39s] 2023-01-19T09:37:53 <WRN> [OTA] Reboot in 5sec
```

### Screenshots

![image](https://user-images.githubusercontent.com/115730895/213393432-16c44f16-2971-42d4-b6c4-c87e0f32f4b7.png)

**Comment by nliaudat** [comment]
Sorry,
I was not in a good day :(

I mean I wanted to disable that function : 

ClassFlowAlignment.cpp:240
```
//no align algo if set to 3 = off
if(References[0].alignment_algo != 3){
    if (!AlignAndCutImage->Align(&References[0], &References[1])) 
    {
        SaveReferenceAlignmentValues();
    }
}// no align
```

**Comment by Slider0007** [comment]
This looks better, but you have to think about what could happen when you are not saving the references anymore. For example, reference infos are used to draw the reference rectangles.

https://github.com/jomjol/AI-on-the-edge-device/blob/782e1c229b3cd478bc18b6a202c3fe5dde6af493/code/components/jomjol_flowcontroll/ClassFlowAlignment.cpp#L242

Issue closed by reverting the code with PR #1876. Your welcome to update and test your code then it can be merged again.
